library/spark/spark-library/src/main/java/uk/gov/gchq/gaffer/spark/operation/dataframe/converter/schema/SchemaToStructTypeConverter.java

```diff
@@ -152,35 +153,40 @@ private void buildSchema() {
             }
             final Set<String> properties = elementDefn.getProperties();
             for (final String property : properties) {
-                // Check if property is of a known type that can be handled by default
-                final String propertyClass = elementDefn.getPropertyClass(property).getCanonicalName();
-                DataType propertyType = getType(propertyClass);
-                if (null != propertyType) {
-                    propertyNeedsConversion.put(property, needsConversion(propertyClass));
-                    structFieldList.add(new StructField(property, propertyType, true, Metadata.empty()));
-                    LOGGER.info("Property {} is of type {}", property, propertyType);
-                } else {
-                    // Check if any of the provided converters can handle it
-                    if (null != converters) {
-                        for (final Converter converter : converters) {
-                            if (converter.canHandle(elementDefn.getPropertyClass(property))) {
-                                propertyNeedsConversion.put(property, true);
-                                propertyType = converter.convertedType();
-                                converterByProperty.put(property, converter);
-                                structFieldList.add(new StructField(property, propertyType, true, Metadata.empty()));
-                                LOGGER.info("Property {} of type {} will be converted by {} to {}",
-                                        property,
-                                        propertyClass,
-                                        converter.getClass().getName(),
-                                        propertyType);
-                                break;
+                if (!ReservedPropertyNames.contains(property)) {
```

Comment:
Please can we make sure this is documented somewhere so users are aware that the reserved properties are skipped.

Reply:
Added documentation to the class describing the new behaviour.